test/index.html

```diff
@@ -3,8 +3,6 @@
 <head>
 	<meta charset="utf-8">
 	<title>jQuery Migrate Test Suite</title>
-	<!-- Ensure IE doesn't fall back into oldIE modes -->
-	<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
```

Comment:
In IE8 there used to be a button next to the URL to switch into IE7 Compatibility View, which people accidentally pressed. I thought they removed it in IE9, but I just checked and it didn't go away until IE10. Since IE10 the only way for any IE10+ browser to be forced into Compat View by the user is to either manually add it in a deeply buried dialog or use Enterprise Mode and have their IT department list it in a config file. That's super unlikely in our unit test scenario.

Bottom line is that you're right, since it *is* possible to push that button in IE9 I'll restore it.